### PR TITLE
Translate more French words into English

### PR DESCRIPTION
**Pull request by sunxiaoguang**
There are some more words can be translated into English

### CODE REVIEW

File: resources/lang/en/general_content.php [hunk 1]
```diff
@@ -142,7 +142,7 @@
     'label_trans_key'                          => 'Label',
     'customer_trans_key'                       => 'Customer',
     'status_trans_key'                         => 'Status',
-    'statu_trans_key'                          => 'Statu',
+    'statu_trans_key'                          => 'Status',
```
Comment (SMEWebify) [inline resources/lang/en/general_content.php]: I cannot accept this line, there is statu and status with an "s" for the plural (line just before) if I maintain, it is duplicates

**Review by SMEWebify — CHANGES_REQUESTED** [review]

File: resources/lang/en/general_content.php (hunk 1 above)
Comment (sunxiaoguang) [reply resources/lang/en/general_content.php]: Hm, But I guess there is no Statu in English, What can we do with this?

Comment (sunxiaoguang) [reply resources/lang/en/general_content.php]: I think we can leave the discussion for future. This line is reverted, please take another look.

Comment (SMEWebify) [reply resources/lang/en/general_content.php]: state or progress would be more appropriate? I validate the pull request and we will see how it is used later,

**Review by SMEWebify — APPROVED** [review]

File: resources/lang/en/general_content.php (hunk 1 above)
Comment (sunxiaoguang) [reply resources/lang/en/general_content.php]: I guess state and status are pretty similar. The only issue is that they are not related like it is in French. 

I can totally understand the meaning of Statu in UI, even though it's not English. Let's keep it as it is.

**Pull request merged**